dev-support/Jenkinsfile

```diff
@@ -49,6 +50,8 @@ pipeline {
     // These tests currently have known failures. Once they burn down to 0, remove from here so that new problems will cause a failure.
     TESTS_FILTER = 'cc,checkstyle,javac,javadoc,pylint,shellcheck,whitespace,perlcritic,ruby-lint,rubocop,mvnsite'
     EXCLUDE_TESTS_URL = "${JENKINS_URL}/job/HBase-Find-Flaky-Tests/job/${BRANCH_NAME}/lastSuccessfulBuild/artifact/excludes"
+    SHALLOW_CHECKS = 'all,-unit,-findbugs' // run by the 'yetus general check'
```

Comment:
Actually, why do we skip `findbugs` entirely? It should be done either as a shallow check or as a deep check, yes?

Reply:
we used to do findbugs just during one of the hadoop checks I think. line 345 prior to this changest.

Reply:
so long as the general check is jdk8 then doing it in general is good. IIRC when we were sharing this jenkinsfile with branch-1 it was in jdk8-hadoop2 because that was jdk8 on both sets of branches.

Reply:
Oh, I see. Thanks for the pointer.

Reading through the nightly code again, I'm back to believing that I'll break branch-1. Let me add back JDK7 to the docker file as well.